packages/client/src/views/SysAdmin/Config/utils.ts

```diff
@@ -113,7 +113,7 @@ export const getCurrencySelectOptions = () => {
 }
 
 export const getFormattedFee = (value: string) => {
-  value = value.replace(/\,/g, '')
+  value = value.replace(/,/g, '')
```

Comment:
Could you make this a new variable so we wouldn't override the input parameter? I know you didn't write it like this initially, but a good place to clean things up a bit :)

Reply:
Yeah. Sure thing.